docs/developer-guide.md

```diff
@@ -48,3 +48,67 @@ const showFlaggedFeature = useShowGrowthbookFeature('flagged-feature'); // Set t
 ### Cleanup
 
 Once a feature flag is no longer necessary, any code that was added (ex: `useShowGrowthbookFeature`) should be removed and the flag in Growthbook can be deleted.
+
+## Database Testing
+
+### pgTap + sqitch tips
+
+#### Run a single test
+
+You can run a single test by calling `pg_prove` directly from the command line like so:
+`pg_prove -v -d \<test_db_name\> \<path-to-test-file\>/file_to_test.sql`
+flags:
+
+- `-v`runs pg_prove in verbose. This allows you to add select statements in the test file & they will show up in the output (helpful for debugging)
+- `-d` sets the target database (must be right before your test database)
+
+The path to the test file can be a glob pattern. So if you want to run all tests in a specific folder, or don't feel like writing our the whole path,
+you could write the command like this (assuming our tests are found at `test/unit/tables/file_test.sql`:
+
+Run all table tests:
+`pg_prove -v -d \<test_db_name\> test/unit/tables/*_test.sql`
+Run a specific test anywhere in the 'unit' directory:
+`pg_prove -v -d \<test_db_name\> test/unit/**/specific_file_test.sql`
```

Comment:
Line breaks and extra lines, otherwise markdown renders everything on the same line

```suggestion
Run all table tests:<br/>
`pg_prove -v -d \<test_db_name\> test/unit/tables/*_test.sql`

Run a specific test anywhere in the 'unit' directory:<br/>
`pg_prove -v -d \<test_db_name\> test/unit/**/specific_file_test.sql`
```